### ISSUE TITLE
Adjust calendar size to the sidebar width

### ISSUE THREAD

**Issue by stracker-phil** [srg-kostyrko/obsidian-journal#44]
The calendar in the sidebar has a static width and does not adjust to the sidebar size.

**Reason:**
The grid template uses a fixed column width of `30px`

**Suggestion:**
Switch to `fr` for grid-column sizing for a dynamic grid size.

**Sample CSS:**
```css
.journal-calendar-view-month.with-week {
    /* old: grid-template-columns: repeat(8, 30px); */
    grid-template-columns: 1.25fr repeat(7, 1fr); 
}

.journal-calendar-view-month {
    ...
    /* old: grid-template-columns: repeat(7, 30px); */
    grid-template-columns: repeat(7, 1fr);
    ...
}
```

<details>
<summary><strong>Result:</strong></summary>

![2024-04-20_23-04-53 (1)](https://github.com/srg-kostyrko/obsidian-journal/assets/2669200/920d35ac-31fc-4cdb-b408-739750d0dc90)

</details>

**Comment by srg-kostyrko** [comment]
good point, thanks for suggestion, will add it

**Comment by srg-kostyrko** [comment]
released in 1.4.0
